src/registration/Register.tsx

```diff
@@ -0,0 +1,223 @@
+        />
+      </div>
+      <div style={{ lineHeight: '170%', marginTop: '0.25rem' }}>
+        <LabeledCheckbox
+          checked={!partOfOrganization}
+          onChange={() => setPartOfOrganization(!partOfOrganization)}
+          disabled={false}
+        >
+          <span style={{ marginLeft: '0.25rem' }}>I am not a part of an organization</span>
+        </LabeledCheckbox>
+      </div>
+      <div style={{ display: 'flex' }}>
+        <div style={{ lineHeight: '170%' }}>
+          <LabelledTextInput
+            value={department}
+            required={partOfOrganization}
+            disabled={!partOfOrganization}
+            onChange={setDepartment}
+            labelStyle={{ display: 'block' }}
+            label="Department"
+          />
+        </div>
+        <div style={{ width: '1rem' }} />
+        <div style={{ lineHeight: '170%' }}>
+          <LabelledTextInput
+            value={title}
+            required={partOfOrganization}
+            disabled={!partOfOrganization}
+            onChange={setTitle}
+            label="Title"
+            labelStyle={{ display: 'block' }}
+          />
+        </div>
+      </div>
+      <h3 style={{ marginTop: '2rem' }}>I am most interested in using Terra to (Check all that apply):</h3>
+      <CheckboxLine>
+        {_.map(
+          (title: string) => {
+            return (
+              <InterestInTerraCheckbox
+                title={title}
+                interestInTerra={interestInTerra}
+                setFunc={setInterestInTerra}
+                key={_.uniqueId('interest_')}
+              />
+            );
+          },
+          [
+            'Collaborate with individuals within my organization',
+            'Collaborate with individuals outside of my organization',
+            'Access data',
+            'Manage datasets',
+            'Launch workflows',
+            'Complete interactive analyses',
+            'Build Tools',
+          ]
+        )}
+      </CheckboxLine>
+      <h3 style={{ marginTop: '2rem' }}>Communication Preferences</h3>
+      <CommunicationPreferencesCheckbox
+        title="Necessary communications related to platform operations"
+        value
+        setFunc={undefined}
+      />
+      <CommunicationPreferencesCheckbox
+        title="Marketing communications including notifications for upcoming workshops and new flagship dataset additions"
+        value={marketingConsent}
+        setFunc={setMarketingConsent}
+      />
+      <div style={{ marginTop: '3rem' }}>
+        <ButtonPrimary disabled={errors || busy} onClick={register}>
+          Register
+        </ButtonPrimary>
+        <ButtonSecondary style={{ marginLeft: '1rem' }} onClick={() => signOut('requested')}>
+          Cancel
+        </ButtonSecondary>
+        {busy &&
+          centeredSpinner({ size: 34, ...{ style: { display: undefined, margin: undefined, marginLeft: '1ex' } } })}
+      </div>
+    </div>
+  );
+};
+export default Register;
```

Comment:
Prefer named exports (`export const Register = ...`) over default exports since they're easier to mock in unit tests and aren't as easy to accidentally alias in imports.